### PR TITLE
file_sys/control_metadata: Amend naming of members

### PR DESCRIPTION
**Pull request by lioncash**
Quite a bit of these were out of sync with Switchbrew (and in some cases entirely wrong). While we're at it, also expand the section of named members. A segment within the control metadata is used to specify maximum values for the user, device, and cache storage max sizes and journal sizes.

These new added member seem to generally be used by the am service (e.g. in IApplicationFunctions' CreateCacheStorage, etc).

### CODE REVIEW

**Comment by bunnei** [comment]
@lioncash please address suggestion ^

File: src/core/file_sys/control_metadata.h [hunk 1]
```diff
@@ -38,23 +38,35 @@ struct RawNACP {
...
     std::array<char, 0x10> version_string;
     u64_le dlc_base_title_id;
-    u64_le title_id_2;
-    u64_le normal_save_data_size;
-    u64_le journal_sava_data_size;
-    INSERT_PADDING_BYTES(0x18);
-    u64_le product_code;
+    u64_le save_data_owner_id;
+    u64_le user_account_save_data_size;
+    u64_le user_account_sava_data_journal_size;
```
[the remark this answers is not in the document]
Comment (lioncash) [reply src/core/file_sys/control_metadata.h]: I have no idea what's being suggested based off the diff.

[the remark this answers is not in the document]
Comment (FearlessTobi) [reply src/core/file_sys/control_metadata.h]: sava -> save

[the remark this answers is not in the document]
Comment (lioncash) [reply src/core/file_sys/control_metadata.h]: Ah, ok. Knew I was missing something obvious

**Pull request merged**